### PR TITLE
cmd/atlas: support sqltool formatters in ent extension

### PR DESCRIPTION
**Pull request by a8m**

### CODE REVIEW

File: cmd/atlas/internal/cmdext/cmdext_test.go [hunk 1]
```diff
@@ -118,21 +119,41 @@ func TestEntLoader_MigrateDiff(t *testing.T) {
...
+	t.Run("AtlasFormat", func(t *testing.T) {
+		dir, err := migrate.NewLocalDir(t.TempDir())
+		require.NoError(t, err)
+		err = d.MigrateDiff(ctx, &cmdext.MigrateDiffOptions{
+			Name: "boring",
+			Dev:  drv,
+			Dir:  dir,
+			To:   []string{"ent://../migrate/ent/schema?globalid=1"},
+		})
+		require.NoError(t, err)
```
Comment (masseelch) [inline cmd/atlas/internal/cmdext/cmdext_test.go:134]: 💯

File: cmd/atlas/internal/cmdext/cmdext.go [hunk 2]
```diff
@@ -296,7 +297,7 @@ func (l EntLoader) MigrateDiff(ctx context.Context, opts *MigrateDiffOptions) er
 	}
 	m, err := entschema.NewMigrate(
 		sql.OpenDB(opts.Dev.Name, opts.Dev.DB),
-		entschema.WithFormatter(migrate.DefaultFormatter),
+		entschema.WithFormatter(dirFormatter(opts.Dir)),
 		entschema.WithGlobalUniqueID(true),
 		entschema.WithDir(opts.Dir),
```
Comment (masseelch) [inline cmd/atlas/internal/cmdext/cmdext.go:302]: I feel like we could have done that inside Ent, right?

**Review by masseelch — APPROVED** [review]

File: cmd/atlas/internal/cmdext/cmdext.go (hunk 2 above)
Comment (a8m) [reply cmd/atlas/internal/cmdext/cmdext.go:302]: Yes, and I prefer to do it there. Let me move this logic there and update the Ent in this repo.

Comment (a8m) [reply cmd/atlas/internal/cmdext/cmdext.go:302]: Sorry no, because of this: 
image

**Pull request merged**